octobot_trading/personal_data/orders/order.py

```diff
@@ -25,8 +25,11 @@
 import octobot_trading.enums as enums
 import octobot_trading.errors as errors
 import octobot_trading.personal_data.orders.states as orders_states
+import octobot_trading.personal_data.orders.order_factory as order_factory
```

Comment:
This is the kind of import that makes me think there's an architecture issue here